### PR TITLE
https://github.com/actions/.github/security/policy

### PR DESCRIPTION
**Pull request by Rhinowiwina**
From 5bfa01d115eb5c9904daa9636c1cd43ace2b33ca Mon Sep 17 00:00:00 2001
From: Rhino <86544472+Rhinowiwina@users.noreply.github.com>
Date: Sat, 28 Aug 2021 18:06:03 -0700
Subject: [PATCH] Update and rename SECURITY.md to
 SECURITY.mdhttps:/github.com/actions/.github

---
 SECURITY.md => SECURITY.mdhttps:/github.com/actions/.github | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 rename SECURITY.md => SECURITY.mdhttps:/github.com/actions/.github (69%)

diff --git a/SECURITY.md b/SECURITY.mdhttps:/github.com/actions/.github
similarity index 69%
rename from SECURITY.md
rename to SECURITY.mdhttps:/github.com/actions/.github
index f0b196f..3960ee8 100644
--- a/SECURITY.md
+++ b/SECURITY.mdhttps:/github.com/actions/.github
@@ -1,3 +1,3 @@
 If you discover a security issue in this repo, please submit it through the [GitHub Security Bug Bounty](https://hackerone.com/github)
-
-Thanks for helping make GitHub Actions safe for everyone.
+ .github/
+in

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Thank you 🙇‍♀ for wanting to create an issue in this repository. Before you do, please ensure you are filing the issue in the right place. Issues should only be opened on if the issue **relates to code in this repository**.
 
 * If you have found a security issue [please submit it here](https://hackerone.com/github)
+* If you have questions about writing workflows or action files, then please [visit the GitHub Community Forum's Actions Board](https://github.community/c/code-to-cloud/github-actions)
 * If you are having an issue or question about GitHub Actions then please [contact customer support](https://help.github.com/en/articles/about-github-actions#contacting-support)
-* If you have questions about to write workflow or action files then please [visit the community forum](https://github.community/t5/GitHub-Actions/bd-p/actions)
```
Comment (stonehead8111) [inline .github/issue_template.md:5]: S

Comment (stonehead8111) [reply .github/issue_template.md:5]: W

Comment (stonehead8111) [reply .github/issue_template.md:5]: W

**Comment by stonehead8111** [comment]
W

**Comment by stonehead8111** [comment]
S

**Review by stonehead8111 — COMMENTED** [review]
Duplicate of #

**Comment by stonehead8111** [comment]
WEQ.DIFF

**Comment by asanpardazzagros-debug** [comment]
main

**Comment by cashpilotthrive-hue** [comment]
Help set up my account

**Pull request closed without merging**